proxy/ProxyClientTransaction.cc

```diff
@@ -66,6 +67,8 @@ ProxyClientTransaction::release(IOBufferReader *r)
   HttpTxnDebug("[%" PRId64 "] session released by sm [%" PRId64 "]", parent ? parent->connection_id() : 0,
                current_reader ? current_reader->sm_id : 0);
 
+  this->decrement_client_transactions_stat();
```

Comment:
Actually, I'm not confident that `ProxyClientTransaction::release` is always called even if some error happened.

Reply:
Wouldn't that imply a memory leak?

Reply:
Good point. We don't see memory leaks on Http1ClientTransaction/Http2Stream so far, so we can say this is always called. Thanks.